src/utils/media-utils.js

```diff
@@ -68,11 +68,18 @@ export const determineMediaDuration = memoizee(async url => {
   await Promise.resolve();
 
   const playerPromise = new Promise((resolve, reject) => {
+    let validUrl;
+    if (!url) {
+      resolve(null);
+    }
+
+    try {
+      validUrl = new URL(url).href;
+    } catch {
+      resolve(null);
+    }
```

Comment:
Just added this to remove the console errors logged while typing an external URL. Basically I think we should treat a missing value the same as a partial value (aka URL while typing).